src/app/public/modules/autonumeric/fixtures/autonumeric.component.fixture.html

```diff
@@ -18,6 +19,7 @@
   <input
     aria-label="Donation amount"
     class="sky-form-control app-template-driven-input"
+    id="donationAmountTemplateDriven"
```

Comment:
Is `id` needed for these?

Reply:
I added `id` to make getting the elements from the spec files more deliberate.